Gemfile.lock

```diff
@@ -288,9 +288,15 @@ GEM
     rb-inotify (0.9.5)
       ffi (>= 0.5.0)
     rdoc (4.2.0)
+<<<<<<< HEAD
```

Comment:
something wrong with this file
I guess there were some problems while rebasing?

hint: you can delete this file and rerun ```bundle``` to generate new one